templates/taxi/car_form.html

```diff
@@ -2,11 +2,11 @@
 {% load crispy_forms_filters %}
 
 {% block content %}
-  <h1>{{ object|yesno:"Update,Create" }} car</h1>
+  <h1>Create driver</h1>
```

Comment:
The heading 'Create driver' appears to be incorrect for this template, which is intended for a car form. Consider changing it to 'Create Car' or a similar title that reflects the purpose of the form.